### PR TITLE
feat: added filter with brand name

### PR DESCRIPTION
**Pull request by Shubhamkr2610**

### CODE REVIEW

File: src/pages/product-page/ProductPage.js [hunk 1]
```diff
@@ -32,7 +36,10 @@ export const ProductPage = () => {
                 <div className='product-wrapper'>
                    
                    {PriceWishFilter.map(item=><ProductCard key={item._id} title={item.title} brand={item.brand} discountedPrice={item.discountedPrice} actualPrice={item.actualPrice} discountoff={item.discountoff} srcimg={item.srcimg} rating= {item.rating} />)}
-                   
+
+
+                   {/* {categoryFilteredProduct.map(item=><ProductCard key={item._id} title={item.title} brand={item.brand} discountedPrice={item.discountedPrice} actualPrice={item.actualPrice} discountoff={item.discountoff} srcimg={item.srcimg} rating= {item.rating} />)}
+                    */}
                 </div>
```
Comment (Rockyraven) [inline src/pages/product-page/ProductPage.js:40]: remove this unwanted code

File: src/components/sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -30,29 +30,32 @@ export const Sidebar = ({ dispatch, state}) => {
...
-                        <div>
-                            <input type="checkbox" id="range-1000" />
-                            <label htmlFor="range-1000">1000- 1500</label>
+                         {/* -----------------------filter by brand -------------- */}
+
+                         <div>
+                            <input type="checkbox" className="input-checkbox" id="nike-collection" checked={state.Nike} onChange={() => dispatch({type: "NIKE_BRAND"})} />
+                            <label htmlFor="nike-collection">Nike</label>
                         </div>
-                        <div>
```
Comment (Rockyraven) [inline src/components/sidebar/Sidebar.jsx:41]: these commented line is really appreciation:
It helps to readability while reviewing

**Review by arjun-computer-geek — APPROVED** [review]
Code is cleaner and readable. Well commented code. And well formatted code. Good file structure.

File: src/pages/product-page/ProductPage.js (hunk 1 above)
Comment (Shubhamkr2610) [reply src/pages/product-page/ProductPage.js:40]: Good pick , Thank you , fixed it .

**Review by Rockyraven — APPROVED** [review]
remove some extra unwanted code
otherwise your code is clean and easy to readalbe

**Pull request merged**